lib/core/utils/to-array.js

```diff
@@ -7,4 +7,17 @@
 axe.utils.toArray = function (thing) {
 	'use strict';
 	return Array.prototype.slice.call(thing);
-};
\ No newline at end of file
+};
+
+
+/**
+ * Creates an array without duplicate values from 2 array inputs
+ * @param  {Array} arr1 First array
+ * @param  {Array} arr2 Second array
+ * @return {Array}
+ */
+axe.utils.uniqueArray = (arr1, arr2) => {
```

Comment:
This is super minor, but it'd be nice if this iterated over an arbitrary number of array inputs.